src/Themes/HackathonTheme/Migrations.cs

```diff
@@ -270,5 +278,741 @@ private void VContainer()
+                )
+                .WithField("Color", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("Color")
+                    .WithEditor("Color")
+                    .WithSettings(new TextFieldSettings
+                    {
+                        Hint = "Applies specified color to the control - it can be the name of material color (for example success or purple) or css color (#033 or rgba(255, 0, 0, 0.5)).",
+                    })
+                )
+                .WithField("Elevation", field => field
+                    .OfType("NumericField")
+                    .WithDisplayName("Elevation")
+                    .WithEditor("Slider")
+                    .WithSettings(new NumericFieldSettings
+                    {
+                        Hint = "Designates an elevation applied to the component between 0 and 24",
+                        Minimum = 0,
+                        Maximum = 24,
+                        DefaultValue = "0",
+                    })
+                )
+                .WithField("Icon", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("Icon")
+                    .WithSettings(new TextFieldSettings
+                    {
+                        Hint = "Designates a specific icon.",
+                    })
+                )
+                .WithField("Type", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("Type")
+                    .WithEditor("PredefinedGroup")
+                    .WithSettings(new TextFieldSettings
+                    {
+                        Hint = "Specify a success, info, warning or error alert. Uses the contextual color and has a pre-defined icon.",
+                    })
+                    .WithSettings(new TextFieldPredefinedGroupEditorSettings
+                    {
+                        Options = new ListValueOption[] { new ListValueOption() {
+                            Name = "Success",
+                            Value = "success"
+                        }, new ListValueOption() {
+                            Name = "Info",
+                            Value = "info"
+                        }, new ListValueOption() {
+                            Name = "Warning",
+                            Value = "warning"
+                        }, new ListValueOption() {
+                            Name = "Error",
+                            Value = "error"
+                        } },
+                    })
+                )
+            );
+        }
+
+        private void VCard()
+        {
+            _contentDefinitionManager.AlterTypeDefinition("VCard", type => type
+                .DisplayedAs("VCard")
+                .Stereotype("Widget")
+                .WithPart("VCard", part => part
+                    .WithPosition("1")
+                )
+                .WithPart("TitlePart", part => part
```

Comment:
For these, try using the extension method WithTitlePart() I added to the extensions